Text Formatting.html

```diff
@@ -187,19 +187,19 @@ <h2><a name="Introduction">Introduction</a></h2>
 Even with proliferation of graphical and voice user interfaces, text remains one
 of the main ways for humans to interact with computer programs and programming
 languages provide a variety of methods to perform text formatting.
-The first thing we do when learning a new programming language is often writing
+The first thing we do when learning a new programming language is often to write
```

Comment:
Is "to" needed here?

Reply:
Not strictly necessary but it sounds more formal to my ears. Happy either way.